src/index.js

```diff
@@ -80,6 +81,12 @@ export default class MediaQuery extends React.Component {
       matches: this._mql.matches
     })
   }
+  
+  componentDidUpdate(_, prevState) {
```

Comment:
@contra I have no idea what the original use case was, just wanted to contribute (and practice writing JS tests in libraries :P)

My reasoning for running this after render was that you might want to, for example, bind/rebind any external DOM events or integrations if the children were changed (for ex. to rebind Google Maps or redraw `<canvas/>`) while I couldn't think of any useful case where you would want the `onChange` to fire before the render.

If you think it should be `componentWillUpdate` then I'm happy to change this, just let me know :)